source/blog/2016-11-02-ovirt-provider-ovn.md

```diff
@@ -204,7 +205,44 @@ If used to set up ovirt-provider-ovn, engine-setup will perform the following ta
 * install ovs/ovn packages
```

Comment:
maybe we should mention that engine-setup handles only the installation of OVN packages on the central server.